src/System.IO.Compression.Brotli/System.IO.Compression.Brotli.csproj

```diff
@@ -12,24 +15,54 @@
     <OutputTypeEx>library</OutputTypeEx>
     <StartupObject />
   </PropertyGroup>
-  <!-- Platform specific properties -->
-  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
+  <PropertyGroup Condition=" '$(Platform)' == 'x64'">
```

Comment:
We use ItemGroup for different actions like(Include, Remove, Exclude, etc..) and PropertyGroup to create a set of user-defined Property elements. https://msdn.microsoft.com/en-us/library/t4w159bs.aspx 
Thank you for recommendation about Choose/When I'll think about it, because for me it isn't really clearer